projects/aws-observability/aws-otel-collector/Makefile

```diff
@@ -1,6 +1,9 @@
 BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
 GIT_TAG:=$(shell cat GIT_TAG)
+GIT_HASH:=$(shell git -C $(BASE_DIRECTORY) rev-parse HEAD)
 HELM_GIT_TAG:=$(shell cat HELM_GIT_TAG)
+IMAGE_TAG=$(GIT_TAG)
```

Comment:
image_tag needs to match whats in git_tag since image is pulled from ecr directly

Reply:
I added a comment to explain the rationale.